test/unit-tests/common/test_image_star.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+    :copyright: Copyright 2017-2019 by the contributors (see AUTHORS file).
+    :license: BSD-2-Clause, see LICENSE for details.
+"""
+
+from sphinxcontrib.confluencebuilder.state import ConfluenceState
+from sphinxcontrib_confluencebuilder_util import ConfluenceTestUtil as _
+from sphinxcontrib.confluencebuilder.builder import ConfluenceBuilder
+import os
+import re
+import unittest
+import tempfile
+import shutil
```

Comment:
(optional) Re-ordering these imports into lexicographical order would be a nicety.